docs/velox-backend-support-progress.md

```diff
@@ -344,25 +343,25 @@ Gluten supports 199 functions. (Drag to right to see all data types)
+| extract                      | extract                | extract               | S      |                          |         |      |       |     |      |       |        | S    | S         |        |         |      |        |          |       |     |        |     |
 | from_unixtime                | from_unixtime          |                       | S      |                          |         |      |       |     | S    |       |        |      |           |        |         |      |        |          |       |     |        |     |
 | from_utc_timestamp           |                        | from_utc_timestamp    | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
 | hour                         | hour                   |                       | S      |                          |         |      |       |     |      |       |        |      | S         |        |         |      |        |          |       |     |        |     |
 | last_day                     |                        | last_day              | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
 | make_date                    |                        | make_date             | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
 | make_dt_interval             |                        |                       |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| make_interval                |                        |                       |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| make_timestamp               |                        |                       |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| make_ym_interval             |                        |                       |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
+| make_interval                | make_interval          | make_interval         | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
```

Comment:
The return type for "make_interval" is `CalendarIntervalType`, which is not supported yet.